src/languages/axapta.js

```diff
@@ -1,20 +1,152 @@
 /*
-Language: Microsoft Axapta (now Dynamics 365)
+Language: Microsoft X++, formerly Axapta, now Microsoft Dynamics 365
 Author: Dmitri Roudakov <dmitri@roudakov.ru>
 Website: https://dynamics.microsoft.com/en-us/ax-overview/
 Category: enterprise
 */
 
 /** @type LanguageFn */
 export default function(hljs) {
+  var BUILT_IN_KEYWORDS = [
+    'anytype',
+    'boolean',
+    'byte',
+    'char',
+    'container',
+    'date',
+    'double',
+    'enum',
+    'guid',
+    'int',
+    'int64',
+    'long',
+    'real',
+    'short',
+    'str',
+    'utcdatetime',
+    'var'
+  ];
+
+  var LITERAL_KEYWORDS = [
+    'default',
+    'false',
+    'null',
+    'true',
+
+  ];
+
+  var NORMAL_KEYWORDS = [
+    'abstract',
+    'as',
+    'asc',
+    'avg',
+    'break',
+    'breakpoint',
+    'by',
+    'byref',
+    'case',
+    'catch',
+    'changecompany',
+    'class',
+    'client',
+    'client',
+    'common',
+    'const',
+    'continue',
+    'count',
+    'crosscompany',
+    'delegate',
+    'delete_from',
+    'desc',
+    'display',
+    'div',
+    'do',
+    'edit',
+    'else',
+    'eventhandler',
+    'exists',
+    'extends',
+    'final',
+    'finally',
+    'firstfast',
+    'firstonly',
+    'firstonly1',
+    'firstonly10',
+    'firstonly100',
+    'firstonly1000',
+    'flush',
+    'for', 
+    'forceliterals',
+    'forcenestedloop',
+    'forceplaceholders',
+    'forceselectorder',
+    'forupdate',
+    'from',
+    'generateonly',
+    'group', 
+    'hint',
+    'if',
+    'implements',
+    'in',
+    'index',
+    'insert_recordset',
+    'interface',
+    'internal',
+    'is',
+    'join',
+    'like',
+    'maxof',
+    'minof',
+    'mod',
+    'namespace',
+    'new',
+    'next',
+    'nofetch',
+    'notexists',
+    'optimisticlock',
+    'order',
+    'outer',
+    'pessimisticlock',
+    'print',
+    'private',
+    'protected',
+    'public',
+    'readonly',
+    'repeatableread',
+    'retry',
+    'return',
+    'reverse',
+    'select',
+    'server',
+    'setting',
+    'static', 
+    'sum',
+    'super',
+    'switch',
+    'this',
+    'throw',
+    'try',
+    'ttsabort',
+    'ttsbegin',
+    'ttscommit',
+    'unchecked',
+    'update_recordset',
+    'using',
+    'validtimestate',
+    'void',
+    'where',
+    'while',
+  ];
+
+  var KEYWORDS = {
+    keyword: NORMAL_KEYWORDS.join(' '),
+    built_in: BUILT_IN_KEYWORDS.join(' '),
+    literal: LITERAL_KEYWORDS.join(' ')
+  };
+
   return {
     name: 'Dynamics 365',
```

Comment:
But is it _'now'_ "Microsoft Dynamics 365" or is it X++ for naming purposes?

Reply:
The language is X++, exclusively used in Dynamics 365. So how about 
Language: Microsoft X++
Description: X++ is a language used in Microsoft Dynamics 365, Dynamics AX, and Axapta.

Reply:
And would update name to be 'X++'

Reply:
Perhaps update the link to

https://docs.microsoft.com/en-us/dynamicsax-2012/developer/x-language-programming-guide

?